nltk/test/unit/files/tweets.20150430-223406.usermention.csv.ref

```diff
@@ -24,10 +24,10 @@ id,text,user_mentions.id,user_mentions.screen_name
 593891106066198529,RT @GloriaDePiero: Nick Clegg is just as responsible for this Govt's failing  plan as David Cameron - he's backed the Tories all the  way,162472533,GloriaDePiero
 593891106590429185,RT @mykilmarnock: Will the person who dropped a Vote SNP badge please call at Kilmarnock police station to collect it. http://t.co/o3sG5B4L…,2787759816,mykilmarnock
 593891106787631104,RT @Markfergusonuk: The Sun’s Twitter worm appears to believe Ed Miliband is winning so far #bbcqt http://t.co/ZgZbSwnZxZ,28079313,Markfergusonuk
-593891106921799680,"@ScottishPleb @AHairyBiker 
+593891106921799680,"@ScottishPleb @AHairyBiker
```

Comment:
it looks like you re-did the trailing whitespace that https://github.com/nltk/nltk/pull/2754 reverted but that's ok, https://github.com/nltk/nltk/pull/2755 should fix the error from here on out 👍

Reply:
You're right! I ran tests afterwards, and they passed, so I think we're in the clear!

Reply:
:+1: